src/v3/src/components/AuthContainer/AuthContainer.tsx

```diff
@@ -14,6 +14,7 @@ import { Box, useMediaQuery } from '@mui/material';
 import classNames from 'classnames';
 import { FunctionComponent, h } from 'preact';
 
+import { version } from '../../../package.json';
```

Comment:
Would it be better to read this off the global or an env variable so we don't bundle package.json?